WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/ui/wizards/WelcomePage.java

```diff
@@ -95,43 +97,47 @@ private Composite createQuestionComposite(final Composite parent) {
 		Image questionIconImage = questionIconImageDescriptor.createImage();
 		questionIcon.setImage(questionIconImage);
 
-		UIUtils.createLabel("   " + labelQuestion, composite);
+		UIUtils.createBoldLabel("   " + labelQuestion, composite);
 
-		final Composite radioButtons = UIUtils.createFullGridedComposite(
-				composite, 1);
-		radioButtons.setLayoutData(new GridData(SWT.BEGINNING, SWT.CENTER,
-				false, true));
-		radioButtons.setLayout(new FillLayout());
-		final Button radioButtonYes = UIUtils.createRadioButton(radioButtons,
-				"Yes");
+		radioButtonYes = UIUtils.createRadioButton(composite, "Yes");
 		radioButtonYes.addSelectionListener(new SelectionListener() {
 
 			@Override
 			public void widgetSelected(SelectionEvent e) {
+				setErrorMessageAndPageComplete(null);
 				removeDynamicContent(parent);
-				dynamicContent = createLoginComposite(parent);
-				setTitle(title + " (1/1)");
+				dynamicContent = createWelcomeComposite(parent);
+				setTitle(title
+						+ " ("
+						+ currentPageNumber
+						+ "/"
+						+ ((RegistrationWizard) getWizard())
+								.getTotalPageNumber() + ")");
 				parent.layout();
 				parent.update();
-				setPageComplete(false);
 			}
 
 			@Override
 			public void widgetDefaultSelected(SelectionEvent e) {
 			}
 		});
+		final Button radioButtonNo = UIUtils.createRadioButton(composite,
+				"No, I have a " + currentRegistration + "-ID");
 
-		radioButtonNo = UIUtils.createRadioButton(radioButtons, "No");
 		radioButtonNo.addSelectionListener(new SelectionListener() {
 
 			@Override
 			public void widgetSelected(SelectionEvent e) {
-				setErrorMessageAndPageComplete(null);
 				removeDynamicContent(parent);
-				dynamicContent = createWelcomeComposite(parent);
-				setTitle(title + " (1/3)");
+				dynamicContent = createLoginComposite(parent);
+				int total = currentRegistration.equals("User") ? ((RegistrationWizard) getWizard())
```

Comment:
I agree that this doesn't look nice, but this was the only solution if I wanted to avoid making project registration classes for user registration wizard. Furthermore, this is the base class (radio button is created in this class for both welcome pages), so you maybe think about moving this to child classes (UserWelcomePage and ProjectWelcomePage) ?